.circleci/config.yml

```diff
@@ -0,0 +1,19 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/python:2.7-jessie
+    steps:
+      - checkout
+      - setup_remote_docker
+      - run:
+          name: Install awscli
+          command: |
+            sudo pip install awscli
+            login="$(aws ecr get-login --region eu-west-1 --no-include-email)"
+            ${login}
+      - run:
+          name: Start container and verify it's working
+          command: |
+            set -x
+            bin/setup && bin/run_tests && bin/stop
```

Comment:
🙋‍♂️ The `set -x` seems to be there to create more visibility on what step is running, right?

What do you think about using `run` commands to achieve the same effect?

```
- run:
    name: Start applications
    command: bin/setup
- run:
    name: Run tests
    command: bin/run_tests
- run:
    name: Clean up running containers
    command: bin/stop
```
?

Reply:
Yes!

Reply:
Yeap, `set -x` was there for that. I'll remove and put back if/when we need to debug.